ocrd_tesserocr/recognize.py

```diff
@@ -1,15 +1,16 @@
 from __future__ import absolute_import
 
 import locale
-locale.setlocale(locale.LC_ALL, 'C') # circumvent tesseract-ocr issue 1670 (which cannot be done on command line because Click requires an UTF-8 locale in Python 3)
 
-from tesserocr import RIL, PSM, PyTessBaseAPI, PyResultIterator, get_languages, iterate_level
-from ocrd.utils import getLogger, concat_padded, xywh_from_points, points_from_xywh, points_from_x0y0x1y1
+from tesserocr import RIL, PSM, PyTessBaseAPI, get_languages
+from ocrd.utils import getLogger, concat_padded, xywh_from_points, points_from_x0y0x1y1
 from ocrd.model.ocrd_page import from_file, to_xml, TextEquivType, CoordsType, GlyphType, WordType
 from ocrd.model.ocrd_page_generateds import TextStyleType, MetadataItemType, LabelsType, LabelType
 from ocrd import Processor, MIMETYPE_PAGE
 from ocrd_tesserocr.config import TESSDATA_PREFIX, OCRD_TOOL
 
+locale.setlocale(locale.LC_ALL, 'C') # circumvent tesseract-ocr issue 1670 (which cannot be done on command line because Click requires an UTF-8 locale in Python 3)
```

Comment:
I am afraid this will break #23 again. The locale needs to be reset **before** importing from tesserocr/Tesseract the first time.

Reply:
Oh right. Should test CLI call as well.

Reply:
Added a simple test, running the CLI and [it passed](https://travis-ci.org/OCR-D/ocrd_tesserocr/jobs/446218522). Won't the locale problem only arise once PyTessAPI is initialized?

Reply:
Nope, that was too quick, fails indeed for Python 3.

Reply:
OK, rearranged it and tests pass for py3+tess4.

Reply:
I cannot see that fix yet. Can you please push?

Reply:
https://github.com/OCR-D/ocrd_tesserocr/commit/cc6b56358f25b79443a009cf87aa62ad16010a8b

Reply:
> cc6b563

That's strange. I can see it here on the web server, but still nothing is available via `git fetch upstream pull/25/head` (where "upstream" is `ssh://git@github.com/OCR-D/ocrd_tesserocr`). Is that normal for GH?

Reply:
sorry, I just forgot to merge FETCH_HEAD (or use pull) for the follow-up changes.